tests/index.js

```diff
@@ -52,19 +49,6 @@ describe('<ContentLoader />:', () => {
       })
     })
 
-    describe('when type is custom', () => {
```

Comment:
You shouldn't remove this test, it ensures the customs loaders will work, but now we have to test using a different way:

1. Insert a custom loader, like I mention above;
2. Test if the children are rendered (probably you haven't to change a lot of code);

If you have any question about test, feel free to call me :)

Reply:
Will be this test ok?
```
const wrapper = mount(
      <ContentLoader>
        <rect x="80" y="17" rx="4" ry="4" width="300" height="13" />
        <rect x="82" y="44" rx="3" ry="3" width="250" height="10" />
        <circle cx="35" cy="35" r="35" />
      </ContentLoader>
  )

  it('should render custom element', () => {
      expect(wrapper.find('rect')).to.have.length(3)
      expect(wrapper.find('circle')).to.have.length(1)
  })
```

Reply:
Sounds good, I think it's ok :)

If you want to organize more, put the `describe('when type is custom', () => {})` within the `describe('Type props are used', () => {})`, you know? But it isn't a problem :)